tests/modules/test_resizeobjects.py

```diff
@@ -60,16 +41,16 @@ def volume_labels():
     return labels
 
 
-def test_resize_by_factor_shrink_image_labels(image_labels, module, object_set, objects, workspace):
-    objects.segmented = image_labels
+def test_resize_by_factor_shrink_image_labels(image_labels, module,  object_set_empty, objects_empty,  workspace_empty):
```

Comment:
There's some incorrect formatting throughout this file. I'll point out the first occurrence and leave the rest to you. 

There should only be one space between parameters.

Reply:
Gah, this is the result of some poorly written `sed` commands...thanks for catching that!